domino_data/data_sources.py

```diff
@@ -451,7 +451,6 @@ def list_objects(self, prefix: str = "", page_size: int = 1000) -> List[_Object]
                 key=key,
             )
             for key in keys
-            if not key.endswith("/")
```

Comment:
can you add a parameter on the method instead? something like
```python
def list_objects(self, prefix: str = "", page_size: int = 1000, include_folders: bool = False) -> List[_Object]:
...
```
so the behavior doesn't change but use can decide to get everything if they wanted to?

Reply:
Yeah, I can do that. It'll be a bit of a painful change because if we want the `page_size` to still be right, we'll need to pass the `include_folders` param to ds-proxy as well but happy to do it

Reply:
oh I meant that you would add the filter client side based on the `include_folders` but I see how that would not respect the page size. Let keep it simple for now then.